README.md

```diff
@@ -182,6 +203,51 @@ func DropHandler(db *sql.DB) http.HandleFunc {
 }
 ```
 
+If you really need global variables or constants, e.g., for defining errors or string constants, put them at the top of your file.
```

Comment:
Missing headline

Reply:
that was on purpose, headline is "avoid global variables", I didn't want to have a headline "do xyz with global variables" below "avoid global variables"

Reply:
I understand, but it looks like it belongs to the above section.

Reply:
GitHub diff was fooling me, sorry for that.